src/lib/i18n/locales/languages.json

```diff
@@ -95,4 +95,8 @@
 		"code": "zh-TW",
 		"title": "Chinese (Traditional)"
 	}
+	{
```

Comment:
This list should be sorted alphabetically according to the code (except is the default en-US at the top).